src/mixer/plugins/volume_mapping.cxx

```diff
@@ -31,8 +31,6 @@
  */
 
 #define _ISOC99_SOURCE /* lrint() */
-#define _GNU_SOURCE /* exp10() */
```

Comment:
Ordering of commits is wrong. With this ordering, we have a commit that we know fails to build - which breaks `git bisect`. Better amend the original commit.